torch/csrc/jit/python_tracer.h

```diff
@@ -27,7 +27,7 @@ Node* preRecordPythonTrace(
     at::ArrayRef<autograd::Variable> inputs,
     std::vector<THPObjectPtr> scalar_args);
 
-std::shared_ptr<Graph> createGraphByTracing(
+std::pair<std::shared_ptr<TracingState>, Stack> createGraphByTracing(
```

Comment:
Changing the return type here makes the API less clear. In many places the only thing used as an output of this is the graph. TracingState is an implementation detail, it should not leak into the python code at all.

Reply:
I am pretty sure TracingState is never used when returned. Only the graph is used. It is still fine to return a pair of Graph, outputs for the few cases where we use the outputs after tracing. But let's stop exposing TracingState and delete the python bindings for it.

Reply:
There are a bunch of places in onnx code where it appears that the trace is returned, but it always ultimately ends up returning the graph. So this can be refactored.